### PR TITLE
Added header function

### PR DESCRIPTION
**Pull request by Michael-Deitz**

### CODE REVIEW

File: scripts/header.js [hunk 1]
```diff
@@ -8,4 +8,17 @@
...
+const parkName = "Cider Falls Park"
+const logoURL = "URL"
+
+
+
+const getHeader = () => {
+    const headerHTML = `
+        <header>
+            <img src="${logoURL}" alt="Park Logo">
+            <h1>${parkName} State Park </h1>
```
Comment (ztrouy) [inline scripts/header.js:11]: Is this supposed to make it say "Cider Falls Park State Park"?

File: scripts/database.js
Comment (ztrouy) [inline scripts/database.js:1]: Is there a reason that `database.js` is modified in this branch?

**Review by ztrouy — CHANGES_REQUESTED** [review]

File: scripts/header.js (hunk 1 above)
Comment (Michael-Deitz) [reply scripts/header.js:11]: its going to pull it from the parkName on line 12

**Review by ztrouy — APPROVED** [review]

**Comment by jaymenelson56** [comment]
Looks awesome!

**Comment by AngRod96** [comment]
looks good!

**Review by AngRod96 — APPROVED** [review]

**Review by jaymenelson56 — APPROVED** [review]

**Pull request merged**